pkg/storage/util/util.go

```diff
@@ -104,3 +105,43 @@ func GetValueFromSecret(sec *corev1.Secret, key string) (string, error) {
 	}
 	return "", fmt.Errorf("secret %q does not contain required key %q", fmt.Sprintf("%s/%s", sec.Namespace, sec.Name), key)
 }
+
+// GenerateStorageName generates a unique name for the storage
+// medium that the registry will use
```

Comment:
Not sure if it should really be a concern but we could "normalize" the bucket name to contain only what  providers consider to be a valid "bucket name":

https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html
https://cloud.google.com/storage/docs/naming

It would make this function more "future proof".

Reply:
The only thing I see missing would be the ending with a dash, I will update to remove a dash at the end